docs/js/ra-doc-exec.js

```diff
@@ -185,232 +214,9 @@ const buildJSCodeBlocksFromTS = async () => {
     applyPreferredLanguage();
 };
 
-// eslint-disable-next-line no-unused-vars
```

Comment:
Did you change anything in the code you've moved? The large diff makes this PR hard to review.

Reply:
No